tests/MomentoTaggedCacheTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+use Illuminate\Support\Facades\Cache;
+
+class MomentoTaggedCacheTest extends BaseTest
+{
+    /**
+     * @covers \Momento\Cache\MomentoTaggedCache
+     */
```

Comment:
You can just add this once on the class declaration instead of on each function.